src/index.html

```diff
@@ -135,7 +135,7 @@ <h2 class="text-info mt-0 mb-2 subtitle">
                                 year on record.
                             </p>
                             <div class="my-3 text-secondary text-right">
-                                <div>2018/01/15</div>
+                                <did>2018/01/15</div>
```

Comment:
Change to div!

Reply:
> Change to div!

fixed in https://github.com/mariab19/web-and-mobile-accessibility/pull/4/commits/538a9a9f2c94aa7fd9e8285e05af69018a6e749a